core/templates/pages/admin-page/navbar/admin-navbar.directive.ts

```diff
@@ -99,7 +99,7 @@ angular.module('oppia').directive('adminNavbar', [
           });
 
           ctrl.logoWhiteImgUrl = UrlInterpolationService.getStaticImageUrl(
-            '/logo/288x128_logo_white.png');
+            '/logo/288x128_logo_white.webp');
```

Comment:
Will this file work with browsers that do not support WebP?

Reply:
Replaced the img tag using this variable with the picture tag.